usr.sbin/adduser/adduser.sh

```diff
@@ -202,6 +203,8 @@ save_config() {
 	echo "msgfile=$msgfile"		>> ${ADDUSERCONF}
 	echo "disableflag=$disableflag" >> ${ADDUSERCONF}
 	echo "uidstart=$uidstart"       >> ${ADDUSERCONF}
+	echo "Zflag=$Zflag"       >> ${ADDUSERCONF}
```

Comment:
I'd maybe call this `zfscreate=no`, just to be a little more obvious from the name

Reply:
Good call.  I changed both Zencrypt and Zcreate to yes/no bools to remove confusion.